lib/ClangImporter/ImportType.cpp

```diff
@@ -1604,6 +1604,7 @@ static ImportedType adjustTypeForConcreteImport(
       // structs, at which point we should really be checking the lifetime
       // qualifiers.
       case clang::Qualifiers::OCL_Strong:
+        break;
       case clang::Qualifiers::OCL_Weak:
         return {Type(), false};
```

Comment:
In order to not change the C/Objective-C behaviour, I think the change should be something like the following:

```
case clang::Qualifiers::OCL_Strong:
case clang::Qualifiers::OCL_Weak:
  if (!isCxx) {
    return {Type(), false};
  }
  break;
```

PD: there might be more changes that might need to be conditional depending on C++.

Reply:
Yeah we can include any necessary objc changes in my branch.